src/slider.css.ts

```diff
@@ -0,0 +1,159 @@
+import {sprinkles} from './sprinkles.css';
+import {vars} from './skins/skin-contract.css';
+import {style, styleVariants} from '@vanilla-extract/css';
+import {mq} from '.';
+
+export const container = style([
+    sprinkles({
+        display: 'flex',
+        flex: 1,
+    }),
+    {
+        width: '100%',
+    },
+]);
+
+export const targetContainer = style([{display: 'inline-block', width: '100%'}]);
+
+export const rangeSlider = style([
+    sprinkles({
+        display: 'flex',
+        position: 'relative',
+        width: '100%',
+    }),
+    {},
+]);
+
+export const sliderBase = style([
+    sprinkles({
+        borderRadius: vars.borderRadii.bar,
+        background: vars.colors.control,
+    }),
+    {
+        WebkitAppearance: 'none',
+        width: '100%',
+        height: 4,
+        outline: 'none',
+        margin: 0,
+        cursor: 'pointer',
+
+        '::-webkit-slider-thumb': {
+            WebkitAppearance: 'none',
+            width: 20,
+            height: 20,
+
+            zIndex: 3,
+            position: 'relative',
+            borderRadius: vars.borderRadii.bar,
+        },
+
+        '@media': {
+            [mq.tabletOrSmaller]: {
+                cursor: 'none',
+            },
+        },
+    },
+]);
+
+export const sliderVariant = styleVariants({
+    ios: [
+        sliderBase,
+        {
+            '::-webkit-slider-thumb': {
```

Comment:
in this case would be the '::-webkit-slider-thumb' ? if so, unfortunately it would not be possible to change this due to the css rules themselves:

<img width="490" alt="Captura de tela 2023-09-04 113950" src="https://github.com/Telefonica/mistica-web/assets/123959276/dd9f438f-181b-4bb5-a983-3d899718c9c6">

Reply:
Oh, my mistake, I thought we had control over that name